### ISSUE TITLE
navbar appears blurry at normal scaling on Edge

### ISSUE THREAD

**Issue by jpsalis** [EdgeTX/edgetx.github.io#21]
I've noticed while using the site that the icons don't look quite right for whatever reason. On Firefox, they look just fine, but perhaps part of the problem stems from the implementation of the navbar itself? It might be best if each button was an actual button, rather than a premade image, same for the text next to the logo.
 On a related note, I've created a pull request that moves the markup for the navbar into default.html, hopefully that should simplify the job if someone more experienced in CSS wants to undertake it.

**Comment by Gemsquash96** [comment]
This is a private email address and definitely not who you are looking for.
Please stop emailing me.

**Comment by jpsalis** [comment]
You're currently "watch"ing the distribution, which gives you automatic emails whenever a change is made. I don't have any control over what you've opted into, but you can unwatch from the email you received, or unwatch from the Github distro itself.

**Comment by phileville** [comment]
Thanks for the other PR.. That was the right way to do it all along.  

The issue with the blurriness is actually related to how some browsers are reducing quality on small images.  I have come to find out it is a common issue that just seems to pop up for a while, and then get fixed again.  Like right now, on my chrome browser it looks ok...a week ago it was blurry until I zoomed in one level, then it was clear again.  Buttons are probabily the right way to go.  I am a still a beginner in CSS, but I am sure I can figure it out when I have some time...

**Comment by jpsalis** [comment]
yeah for sure. If I get time, I'll see if I can take a look at making some styling for buttons as well, I know some basic stuff in CSS but nothing too crazy.
